emission/tests/storageTests/TestTokenQueries.py

```diff
@@ -168,24 +168,24 @@ def test_run_script_show(self):
         esdt.insert({'token':'y'})
         esdt.insert({'token':'z'})
         sp = subprocess.run(["python3", "bin/auth/insert_tokens.py", "--show"], capture_output=True)
-        # The first message is displayed when we run tests locally
+        # The first message is displayed when we run tests locally with the emission environment; we are now setting DB_HOST to `db` by default
```

Comment:
this is true only in dockerized environments

Reply:
Right, I see the difference now in the testing environment I was using.